packages/cli/src/lib/cmds/play.ts

```diff
@@ -161,12 +137,43 @@ class Play {
   }
 }
 
-export async function play(config: Config, parsedArgs: PlayArgs,
-    log: Log = new ConsoleLog('play'), prompt: Prompt = new InquirerPrompt()): Promise<boolean> {
-  const jdkHelper = new JdkHelper(process, config);
-  const androidSdkTools = await AndroidSdkTools.create(process, config, jdkHelper, log);
-  const gradleWrapper = new GradleWrapper(process, androidSdkTools);
-  const googlePlay = new GooglePlay(gradleWrapper);
+/**
+  * Validates that the service account JSON file exists.
+  * @param {string | undefined} path - The path the the JSON file.
+  * @return {boolean} Whether or not the JSON file exists.
+  */
+function validServiceAccountJsonFile(path: string | undefined): boolean {
+  if (path == undefined) {
+    // Log an error
```

Comment:
Are we missing a TODO here?
```suggestion
    //TODO: Log an error
```